.readthedocs.yml

```diff
@@ -1,11 +1,16 @@
-requirements_file: doc/requirements.txt
+version: 2
 
-build:
-    image: latest
+sphinx:
+  configuration: doc/conf.py
 
 python:
-    version: 3.8
+  install:
+    - requirements: doc/requirements.txt
+    - method: pip
+      path: .
+  system_packages: true
```

Comment:
is this necessary?

Reply:
Originally saw it being used in the lightning repo. But removed as it's being deprecated!